cmd/prysmctl/withdrawal/withdrawal.go

```diff
@@ -0,0 +1,188 @@
+package withdrawal
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/fs"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/logrusorgru/aurora"
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/v3/beacon-chain/rpc/apimiddleware"
+	"github.com/prysmaticlabs/prysm/v3/io/prompt"
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+	"go.opencensus.io/trace"
+)
+
+const (
+	apiPath = "/eth/v1/beacon/pool/bls_to_execution_changes"
+)
+
+func setWithdrawalAddresses(c *cli.Context, r io.Reader) error {
+	ctx, span := trace.StartSpan(c.Context, "withdrawal.setWithdrawalAddress")
+	defer span.End()
+	beaconNodeHost := "http://localhost:3500"
+	if c.String(BeaconHostFlag.Name) != "" {
+		beaconNodeHost = c.String(BeaconHostFlag.Name)
+	}
+	u, err := url.ParseRequestURI(beaconNodeHost)
+	if err != nil {
+		return errors.Wrap(err, "invalid format, unable to parse url")
+	}
+	if u.Scheme == "" || u.Host == "" {
+		return fmt.Errorf("provided url %s is not in the format of http(s)://host:port", beaconNodeHost)
+	}
+	foundFilePaths, err := findWithdrawalFiles(c.String(FileFlag.Name))
+	if err != nil {
+		return errors.Wrap(err, "failed to find withdrawal files")
+	}
+	if len(foundFilePaths) == 0 {
+		return errors.New("no compatible files were found")
+	}
+	au := aurora.NewAurora(true)
+	fmt.Println(au.Red("===============IMPORTANT==============="))
+	if !c.Bool(SkipPromptsFlag.Name) {
+		fmt.Println(au.Red("Please read the following carefully"))
+	}
+	fmt.Print("This action will allow the partial withdraw of amounts over the 32 staked eth in your active validator balance. \n" +
+		"You will also be entitled to the full withdrawal of the entire validator balance if your validator has exited. \n" +
+		"The partial or full withdrawal of the validator balance may require several days of processing. \n" +
+		"Please navigate to our website and make sure you understand the full implications of setting your withdrawal address. \n")
+	fmt.Println(au.Red("THIS ACTION WILL NOT BE REVERSIBLE ONCE INCLUDED. "))
+	fmt.Println(au.Red("You will NOT be able to change the address again once changed. "))
```

Comment:
Why is this outside of the `SkipPromptsFlag` check above?

Reply:
I was thinking it should print anyways but I can hide it if it seems like a better experience